### PR TITLE
style: fix no-useless-return rule

### PR DESCRIPTION
**Pull request by abdonrd**
Fixed automatically with the ESLint `--fix` option.

Continue with #1343.

### CODE REVIEW

File: packages/app/src/webapp/cli.ts [hunk 1]
```diff
@@ -138,7 +138,7 @@ const handleQueuedInput = async (nextBlock: HTMLElement) => {
   const queuedInput = disableInputQueueing()
 
   if (!queuedInput) {
-    return
+
```
Comment (myan9) [inline packages/app/src/webapp/cli.ts]: I believe we can do better here

File: packages/app/src/webapp/util/inject.ts [hunk 2]
```diff
@@ -89,7 +89,7 @@ export const injectCSS = (file: StylesheetSpec): void => {
  */
 export const uninjectCSS = ({ key }): void => {
   if (isHeadless()) {
-    return
+
```
Comment (myan9) [inline packages/app/src/webapp/util/inject.ts]: can we merge the branch into `if(!isHeadless())` ?

Comment (myan9) [reply packages/app/src/webapp/util/inject.ts]: tekton2graph.ts looks good, but we can do better in the other file. Thanks!

File: packages/app/src/webapp/cli.ts (hunk 1 above)
Comment (abdonrd) [reply packages/app/src/webapp/cli.ts]: Sure, but as the first comment says, it was fixed automatically with the ESLint `--fix` option.

Anyway, updated!

File: packages/app/src/webapp/cli.ts [hunk 3]
```diff
@@ -137,9 +137,7 @@ export const pasteQueuedInput = (value: string) => {
 const handleQueuedInput = async (nextBlock: HTMLElement) => {
   const queuedInput = disableInputQueueing()
 
-  if (!queuedInput) {
-    return
-  } else if (nextBlock && queuedInput.length > 0) {
+  if (nextBlock && queuedInput.length > 0) {
```
Comment (myan9) [inline packages/app/src/webapp/cli.ts]: Thanks for updating! We may still need the guard here. What about ```if(nextBlock && queuedInput && queuedInput.length > 0)```

Comment (abdonrd) [reply packages/app/src/webapp/cli.ts]: Right!

File: plugins/plugin-k8s/src/lib/controller/status.ts [hunk 4]
```diff
@@ -104,7 +104,7 @@ const headerRow = (opts: IHeaderRow, kind?: string): Row => {
 function handleError (err: CodedError): CodedError {
   if (err.code === 404) {
     // e.g. no crds in this cluster
-    return
+
```
Comment (myan9) [inline plugins/plugin-k8s/src/lib/controller/status.ts]: merge the branches? thanks!

Comment (myan9) [reply plugins/plugin-k8s/src/lib/controller/status.ts]: ^^^ as well as in reverse-i-search.ts. thanks!

**Pull request merged**